src/SnoopCompile.jl

```diff
@@ -1,28 +1,31 @@
 module SnoopCompile
 
-using Serialization, OrderedCollections
-using Core: MethodInstance, CodeInfo
+using SnoopCompileCore
+export @snoopc
 
-export timesum  # @snoopi and @snoopc are exported from their files of definition
+using SnoopCompileAnalysis
 
-const anonrex = r"#{1,2}\d+#{1,2}\d+"         # detect anonymous functions
-const kwrex = r"^#kw##(.*)$|^#([^#]*)##kw$"   # detect keyword-supplying functions
-const kwbodyrex = r"^##(\w[^#]*)#\d+"         # detect keyword body methods
-const genrex = r"^##s\d+#\d+$"                # detect generators for @generated functions
-const innerrex = r"^#[^#]+#\d+"               # detect inner functions
+# needed for snoopc
+using SnoopCompileAnalysis: parcel, read, write, parse_call, format_userimg
```

Comment:
This doesn't make sense to me. These functions are not used anywhere, how can they be "needed" for snoopc?

Reply:
Formerly we told people to call `SnoopCompile.parcel(data)`. This is compatibility glue.